### PR TITLE
Resolve issues with NetApp E-Series LunMapping module

### PR DESCRIPTION
**Pull request by lmprice**
##### SUMMARY
Resolve issues with the Lun Mapping module, including:

* Remove a default value for the LUN number
* Remove code that is provided by module_utils
* Propagate validate_certs variable to the API requests

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
netapp_e_lun_mapping

##### ANSIBLE VERSION

```
ansible 2.4.0 (mapping a3097a8600) last updated 2017/08/16 18:52:12 (GMT +000)
  config file = /home/ansible.cfg
  configured module search path = [u'/home/lib/ansible/modules']
  ansible python module location = /home/lib/ansible
  executable location = /home/bin/ansible
  python version = 2.7.13 (default, May  1 2017, 22:44:36) [GCC 4.9.2]
```

##### ADDITIONAL INFORMATION
N/A

### CODE REVIEW

**Comment by lmprice** [comment]
@hulquest @broncofan Can you review this, please?

File: lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py [hunk 1]
```diff
@@ -96,57 +76,24 @@
...
 from ansible.module_utils.api import basic_auth_argument_spec
-from ansible.module_utils.pycompat24 import get_exception
-from ansible.module_utils.urls import open_url
-
 from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.six.moves.urllib.error import HTTPError
+from ansible.module_utils.netapp import request, eseries_host_argument_spec
+from ansible.module_utils.pycompat24 import get_exception
 
 HEADERS = {
```
Comment (hulquest) [inline lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py:83]: As a future improvement, should we add HEADERS to the netapp util module (and give it a self describing name like HTTP_JSON_HEADERS)?  Requests might already have a static for this.

File: lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py [hunk 2]
```diff
@@ -96,57 +76,24 @@
...
-
-def get_host_and_group_map(module, ssid, api_url, user, pwd):
+def get_host_and_group_map(module, ssid, api_url, user, pwd, validate_certs):
     mapping = dict(host=dict(), group=dict())
 
     hostgroups = 'storage-systems/%s/host-groups' % ssid
     groups_url = api_url + hostgroups
     try:
-        hg_rc, hg_data = request(groups_url, headers=HEADERS, url_username=user, url_password=pwd)
+        hg_rc, hg_data = request(groups_url, headers=HEADERS, url_username=user, url_password=pwd,
```
Comment (hulquest) [inline lib/ansible/modules/storage/netapp/netapp_e_lun_mapping.py:95]: Future improvement: Along the same line of thought as above, should we rename request to something like santricity_request?   Just trying to improve the readability.

**Comment by hulquest** [comment]
Added a few comments about things I would consider future improvements.  Outside of those this change looks good.  
shipit

**Pull request merged**